docs/daily-journal.md

```diff
@@ -43,3 +43,7 @@ Let's unpack this alias:
 * `$ZK_NOTEBOOK_DIR` is set to the absolute path of the current [notebook](notebook.md) when running an alias. Using it allows you to run `zk daily` no matter where you are in the notebook folder hierarchy.
 * We need to use double quotes around `$ZK_NOTEBOOK_DIR`, otherwise it will not be expanded.
 
+If you want to edit today's note, simply use 
+```sh
+$ zk edit journal/daily
```

Comment:
Thanks for the PR! As an alias is set (above), the command will actually be `zk daily`.

It's mentioned on line 33, but it helps make it clearer to put it in here again. But yes, it should be `zk daily` not `zk edit journal/daily`.

Reply:
Thanks, I did not know that. PR edited accordingly !